pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md

```diff
@@ -65,7 +65,8 @@ Meta tag data specifies that the ad meets the AMP Story format, sets the CTA but
 </html>
 [/sourcecode]
 
-The `amp-cta-type` tag must include one of the [available options](#call-to-action-button-text-enum) for the CTA Button text enum. This ensures a consistent user experience for AMP Story readers.
+The `amp-cta-type` tag can be chosen from one of the [available options](#call-to-action-button-text-enum) for the CTA Button text value. This is recommended, as AMP will then automatically localize the text for you when appropriate.
+Alternatively, if you wish to handle localization for yourself, you can include your own custom text for this value.
```

Comment:
Nit, maybe update to:

Choosing the `amp-cta-type` tag from the available CTA Button text options is recommended. AMP will automatically localize predefined options when appropriate. 

Custom text is allowed, but you will need to implement your own localization.

Reply:
Done.